docs/guidance/EnterpriseScaleAnalytics-SelfService.md

```diff
@@ -0,0 +1,117 @@
+# Scaling and Self-Service in Enterprise-Scale Anayltics
+
+Efficient scaling within an Enterprise Data Platform is a much-desired goal of many organizations and IT organizations as business units should be enabled to build their own (data) solutions and applications that fulfill their requirements and needs. However. achieving this goal is a serious challenge as many existing data platforms are not built around the core concepts of scalability and decentralized ownership. This is not only true from an architectural standpoint, but also becomes noticeable when looking at the team structure within many existing data platforms.
+
+## Introduction
+
+Today, many enterprises have built large data platform monoliths around the concept of a single Azure Data Lake Gen2 account and potentially even a single storage container. In addition, a single Azure subscription is most often used for all data platform related tasks and the concept of subscription level scaling is absent in most architectural patterns. This can hinder continued adoption of Azure if users run into any of the [well-know Azure subscription or service-level limitations](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits). Even though some of the constraints are soft limits, hitting these can still have a significant impact within a data platform that should be avoided at best.
+
+Also, many organizations have organized their data platform architecture and teams around functional responsibilities and pipeline stages that need to be applied to the various data assets of an organization. As a result, specialized teams within a data platform own an orthogonal solution such as ingestion, cleansing, aggregation or serving. This organizational and architectural concept leads to a dramatic loss of velocity, because when data consumers on the serving layer require new data assets to be onboarded or functional changes to be implemented for a specific data asset the following processes will need to be followed o safely roll out these changes:
+
+- The Data Consumer must submit a ticket to the functional teams being responsible for the respective pipeline stages.
```

Comment:
I get the intent of these points that we want to highlight the complexity of making a change and the impact to velocity. However, the orchestration involves engaging a single team (let's say data warehouse dev team). The data warehouse team then does the analysis, engages various teams, does the design, build and test etc. Along the way, they (data warehouse dev team) may engage multiple teams each of whom must sign-off on the change before it makes it to production. Can we consider rewording these points? Would you be open to that?

Reply:
Sure. We can do that. I think depending on the setup it can actually vary depending on the setup on the customer side. Hence, whatever we list here, it may deviate with the setup that customers have.